build/NuGetAdditionalFiles/RunCsc

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+THISDIR=$(dirname $0)
+
+dotnet $THISDIR/csc.exe "$@"
```

Comment:
:memo: Non-Windows machines are the places where missing trailing newlines caused problems for some cases in the past. While the other cases of this in the PR are 🙁 for me, this one stands out as a possible problem.